syntax = "proto3";

package examplemsg;

message ExamplePayload {
  string name = 1;
  int32 value = 2;
}
